Fix PR's branch

coltskeen opened the pull request:
Why:
So that the tests will run

Changes:
- switched PR branch from "master" to "main"

Closes #4

coltskeen committed:
Fix PR's branch

Why:
So that the tests will run

Changes:
- switched PR branch from "master" to "main"

Closes #4

ambergkim reviewed (changes requested):
@Colt Skeen Looks like we're missing the .github folder which should contain the ci run and we put the ci.yml in the root. The file structure should be
```
root
    .github
        workflows
            ci.yml
```

coltskeen committed:
Fixed file structure

Why:
- Ci.yml was in the root instead of the .github folder

Changes:
- added .github folder with workflow subfolder
- moved ci.yml to workflow subfolder

closes #4

coltskeen committed:
Fixing linting errors

Why:
- linting tests failed with various recommendations and errors
- believe it was an error with my package-lock.json

Changes:
- deleted node_modules
- deleted package-lock.json
- made linting recommended syntax adjustments to handler.js and hello-world
- ran npm install and npm install tape --save-dev again

Closes #4

ambergkim commented:
Fixed here: https://github.com/trellistk/docs-website/pull/6
